packages/dx-react-grid-material-ui/src/templates/column-chooser/item.test.jsx

```diff
@@ -72,4 +72,21 @@ describe('Item', () => {
     expect(tree.props().data)
       .toMatchObject({ a: 1 });
   });
+
+  it('should process the disabled prop', () => {
+    const tree = shallow((
+      <Item
+        {...defaultProps}
+        disabled
+      />
+    ));
+    expect(tree.find(ListItem).props())
+      .toMatchObject(expect.objectContaining({
```

Comment:
`expect.objectContaining` is not required along with `.toMatchObject`. You can pass your required object directly.